.gitignore

```diff
@@ -6,4 +6,6 @@ dist
 !docs/build
 /.idea
 coverage
-yarn.lock
\ No newline at end of file
+yarn.lock
+.vscode
```

Comment:
This might not be appreciated by vscode users who like to share their config.

Reply:
Editor configurations typically are not checked in. unless I'll be checking in my .idea folder from Jetbrains ;)